test/E2E/setupE2Etests.ps1

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+using namespace System.Runtime.InteropServices
+
+$IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
```

Comment:
Does this need to run in Windows PowerShell?  Otherwise why not just `$IsWindows`?

Reply:
We've been keeping the project working on Windows PowerShell.

Reply:
Seems odd since we currently only support PSCore in the language worker

Reply:
changed this to only run in PSCore. 😎